demos/serverless/src/handlers.js

```diff
@@ -87,6 +87,45 @@ function getNotificaionsConfig() {
 }
 
 // ===== Join or create meeting ===================================
+exports.createMeeting = async(event, context, callback) => {
+  var response = {
+    "statusCode": 200,
+    "headers": {},
+    "body": '',
+    "isBase64Encoded": false
+  };
+
+  if (!event.queryStringParameters.title) {
+    response["statusCode"] = 400;
+    response["body"] = "Must provide title";
+    callback(null, response);
+    return;
+  }
+  const title = event.queryStringParameters.title;
+  const region = event.queryStringParameters.region || 'us-east-1';
+  let meetingInfo = await getMeeting(title);
+  if (!meetingInfo) {
+    const request = {
+      ClientRequestToken: uuid(),
+      MediaRegion: region,
+      NotificationsConfiguration: getNotificaionsConfig(),
```

Comment:
Let's fix the typo in getNotificationsConfig()

Reply:
Oh yes, will fix it